testutil/logger.go

```diff
@@ -0,0 +1,59 @@
+package testutil
+
+import (
+	"os"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus/hooks/test"
+)
+
+// Opt is a function that will modify the logger used
+type Opt func(l *logrus.Logger)
+
+// TestLogger will build a logger that is useful for debugging
+// it respects levels configured by the 'LOG_LEVEL' env var.
+// It takes opt functions to modify the logger used
+func TestLogger(t *testing.T, opts ...Opt) (*logrus.Entry, *test.Hook) {
```

Comment:
I feel personally there have been few times I'd need the log hook when building a test logger. In terms of brevity like your example, maybe there could be an option that just returned a log entry without the hook?